cardinal/system/system_meat_puppet_system.go

```diff
@@ -0,0 +1,319 @@
+	var exisingPlayer component.Player
+	err := cardinal.NewSearch().Entity(filter.Exact(filter.Component[component.Player]())).
+		Each(world, func(id types.EntityID) bool {
+			player, err := cardinal.GetComponent[component.Player](world, id)
+			if err != nil {
+				if isDevelopmentMode() {
+					logger.Errorf("\033[31mError getting Player Component: %v\033[0m", err)
+				}
+				return true
+
+			}
+
+			if player.PlayerID == uint32(pEID) {
+				exisingPlayer = *player
+				return false
+			}
+
+			return true
+		})
+
+	return exisingPlayer, err
+}
+
+func handleAlias(tokens []string, playerID uint32) (err uint8) {
+	vrb := ts.GetActionType(tokens[0])
+	var e uint8
+	if vrb == enums.ActionTypeInventory {
+		// HERE GOES INVENTORY FROM INVENTORY SYSTEM
+		if isDevelopmentMode() {
+			logger.Infof("\033[35m---->HANDLE ALIAS: NOW SHOULD BE GOING TO INVENTORY FROM INVENTORY SYSTEM\033[0m")
+		}
+		e = 0
+	} else if vrb == enums.ActionTypeLook {
+		// HERE GOES STUFF FROM LOOK SYSTEM
+		if isDevelopmentMode() {
+			logger.Infof("\033[35m---->HANDLE ALIAS: NOW SHOULD BE GOING TO STUFF FROM LOOK SYSTEM\033[0m")
+		}
+		e = 0
+	}
+	return e
+}
+
+func handleVerb(tokens []string, playerID uint32) (err uint8) {
+	vrb := ts.GetActionType(tokens[0])
+	var e uint8
+	var resultStr string
+
+	//cmdData := ts.FishTokens(tokens) ---> Not used YET
+	if vrb == enums.ActionTypeLook || vrb == enums.ActionTypeDescribe {
+		if isDevelopmentMode() {
+			logger.Infof("\033[35m---->HANDLE VERB: NOW SHOULD BE GOING TO STUFF FROM LOOK SYSTEM\033[0m")
+		}
+		e = 0
+	} else if vrb == enums.ActionTypeTake {
+		if isDevelopmentMode() {
+			logger.Infof("\033[35m---->HANDLE VERB: NOW SHOULD BE GOING TO TAKE FROM INVENTORY SYSTEM\033[0m")
+		}
+		e = 0
+	} else if vrb == enums.ActionTypeDrop {
+		if isDevelopmentMode() {
+			logger.Infof("\033[35m---->HANDLE VERB: NOW SHOULD BE GOING TO DROP FROM INVENTORY SYSTEM\033[0m")
+		}
+		e = 0
+	} else {
+		if isDevelopmentMode() {
+			logger.Infof("\033[35m---->HANDLE VERB: NOW SHOULD BE GOING TO ACT FROM ACTION SYSTEM\033[0m")
+		}
+		e, resultStr = 0, "testing"
+		if isDevelopmentMode() {
+			logger.Infof("\033[35m---->HANDLE VERB:resultStr: %s\033[0m", resultStr)
+		}
+	}
+	return e
+}
+
+func insultMeat(cErr uint8, badCmd string) string {
+	var eMsg string
+	if cErr == constants.ER_PR_TK_CX {
+		eMsg = "WTF, slow down cowboy, your gonna hurt yourself"
+	} else if cErr == constants.ER_PR_NOP || cErr == constants.ER_PR_TK_C1 {
+		eMsg = "Nope, gibberish\n" +
+			"Stop breathing with your mouth."
+	} else if cErr == constants.ER_PR_ND || cErr == constants.ER_DR_ND {
+		eMsg = "Go where pilgrim?"
+	} else if cErr == constants.ER_DR_NOP {
+		eMsg = "Go " + badCmd + " is nowhere I know of bellend"
+	} else if cErr == constants.GO_NO_EXIT {
+		eMsg = "Can't go that away " + badCmd
+	}
+	return eMsg
+}
+
+func ProcessCommandsTokens2(Tokens []string, PlayerID uint32) error {
```

Comment:
why process command tokens 2?

Reply:
this was for the test file as the first one receives the world context and we need that for the messages.